sdk/servicebus/service-bus/README.md

```diff
@@ -85,90 +71,100 @@ The following sections provide code snippets that cover some of the common tasks
+using the [getSender](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#getsender-string-)
+function. This gives you a sender which you can use to [send](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/sender#send-servicebusmessage-)
+messages.
 
-You can also use the [sendBatch](https://docs.microsoft.com/en-us/javascript/api/@azure/service-bus/sender#sendbatch-sendablemessageinfo---) method to send multiple messages using a single call.
+You can also use the [sendBatch](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/sender#sendbatch-servicebusmessagebatch-)
+method to send multiple messages using a single call.
 
 ```javascript
-const queueClient = serviceBusClient.createQueueClient("my-queue");
-const sender = queueClient.createSender();
+const sender = serviceBusClient.getSender("my-queue");
 await sender.send({
   body: "my-message-body"
 });
-await sender.sendBatch([
-  { body: "my-message-body-1" },
-  { body: "my-message-body-2" },
-  { body: "my-message-body-3" }
-]);
+
+const batch = await sender.createBatch();
+
+// NOTE: tryAdd() returns false if the message could not
+// be added because the batch is at capacity.
+batch.tryAdd({ body: "my-message-body-1" });
+batch.tryAdd({ body: "my-message-body-2" });
+batch.tryAdd({ body: "my-message-body-3" });
+
+await sender.sendBatch(batch);
 ```
 
 ### Receive messages
 
-Once you have created an instance of a `QueueClient` or `SubscriptionClient` class, create a receiver
-using the [createReceiver](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/queueclient#createreceiver-receivemode-) function.
+Once you have created an instance of a `ServiceBusClient` class, you can get a `Receiver`
+using the [getReceiver](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#getreceiver-string--string-) function.
 
 ```javascript
-const queueClient = serviceBusClient.createQueueClient("my-queue");
-const receiver = queueClient.createReceiver(ReceiveMode.peekLock);
+const receiver = serviceBusClient.getReceiver("my-queue", "peekLock");
 ```
 
 You can use this receiver in one of 3 ways to receive messages:
 
 #### Get an array of messages
 
-Use the [receiveMessages](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/receiver#receivemessages-number--number-) function which returns a promise that
+Use the [receiveBatch](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/receiver#receivebatch-number--receivebatchoptions-) function which returns a promise that
 resolves to an array of messages.
 
 ```javascript
-const myMessages = await receiver.receiveMessages(10);
+const myMessages = await receiver.receiveBatch(10);
 ```
 
-#### Register message handler
+#### Subscribe using a message handler
+
+Use the [subscribe](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/receiver#subscribe-messagehandlers--subscribeoptions-)
```

Comment:
Use the subscribe -> Use the subscribe method?

Reply:
Hahah, yes. :|

Reply:
```suggestion
Use the [subscribe](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/receiver#subscribe-messagehandlers--subscribeoptions-) method
```